packages/core/src/Cipher.ts

```diff
@@ -1,21 +1,43 @@
 import { Service } from 'typedi';
-import { AES, enc } from 'crypto-js';
+import { createHash, createCipheriv, createDecipheriv, randomBytes } from 'crypto';
```

Comment:
Maybe not for this commit but do you think we should be moving the code base to the `node:` namespace node imports?

Reply:
yeah. I think we should migrate all node imports in a single PR at some point.